OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -395,4 +395,13 @@ public interface IRemoveFrozenActor
 
 	public interface IRulesetLoaded<TInfo> { void RulesetLoaded(Ruleset rules, TInfo info); }
 	public interface IRulesetLoaded : IRulesetLoaded<ActorInfo>, ITraitInfoInterface { }
+
+    public interface IResource
+    {
+        string ResourceName { get; }
```

Comment:
This wasn't supposed to be in this patch, but I was thinking they weren't because they might collide. I don't really care though. I've pulled this out of this patch though.